app/services/metrics/interval_resolver.rb

```diff
@@ -0,0 +1,34 @@
+module Metrics
+  class IntervalResolver < BaseService
+    def initialize(value)
+      @value = value
+    end
+
+    def call
+      matching_interval || upper_interval
+    end
+
+    private
+
+    def matching_interval
+      ranges.find { |_interval, hour| @value < hour }&.first&.to_s
```

Comment:
`hour` should be renamed to something more generic, right? It can refer to time or PR size.

Reply:
Absolutely! Thank you for noticing it!